public/app/plugins/datasource/loki/language_provider.ts

```diff
@@ -87,6 +87,7 @@ export default class LokiLanguageProvider extends LanguageProvider {
 
     this.datasource = datasource;
     this.labelKeys = [];
+    this.logLabelOptions = [];
```

Comment:
Do we still need this property? I think its usage was removed in https://github.com/grafana/grafana/pull/30351

Reply:
Yes you are right. I didn't realize that we are not using it anymore at all. So I have removed it. 

Also, I noticed, that the try-catch block in `fetchLogLabels` and `fetchLabelValues` is catching only "slice" errors, as request errors are console.error in Loki's request method. So I have changed and simplified this, so "slice" errors are not thrown anymore.
![image](https://user-images.githubusercontent.com/30407135/110824755-a382e500-8293-11eb-9811-3615e8906424.png)

Reply:
Nice one!